src/js/index.js

```diff
@@ -35,7 +35,7 @@ store.subscribe(throttle(model.update, 16));
 var ui = require('./components');
 
 // name model Scene
-model.Scene.name = 'Scene';
+model.Scene._name = 'Scene';
```

Comment:
Apologies for including this here. I realized it might cause knock on effects only after I pushed it. 

The reason I moved `name` to `_name` and exposed a method instead is because users can rename marks within the interface (double click on the name to make it a contenteditable). Users are likely to include spaces in names, but this is illegal in the Vega specification. So the `_name` member holds the exact value a user specified, and the function returns it with spaces replaced with a delimiter while exporting. 

If this is problematic, we can also introduce a `displayName` which is what Lyra 1 used, but I was hoping to avoid the unnecessary additional variable, 

As to why we're treating Scene differently? I'm not exactly sure. This particular line was already in the `lyra2` branch, and this commit just updated it to work with the renamed field. I can expose explicit separate getter and setter.